wmn-data.json

```diff
@@ -3809,6 +3809,15 @@
         "known" : ["notabug", "hp", "zPlus"],
         "cat" : "coding"
        },
+       {
+        "name" : "Noplace",
+        "uri_check" : "https://api.nospace.app/api/v1/profiles/username/availability?username={account}",

```

Comment:
is there any place where we can look on the web (without the noplace app) and find noplace user profiles?

Reply:
Noplace doesn't have a web app, so unfortunatly there is nowhere you can find noplace profiles without the noplace app